### PR TITLE
framework/media: Remove unnecessary code

### PR DESCRIPTION
**Pull request by junmin-kim**
Remove code because header parsing is already implemented

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [c3596c774e2d5de113b6b81ebc065a3f4831990d] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [c3596c774e2d5de113b6b81ebc065a3f4831990d] - Code Rule Check (C++) OK.

File: framework/src/media/InputHandler.cpp [hunk 1]
```diff
@@ -373,9 +373,6 @@ size_t InputHandler::getDecodeFrames(unsigned char *buf, size_t *size)
 	unsigned short channels = 0;
 
 	if (mDecoder->getFrame(buf, size, &sampleRate, &channels)) {
-		/* TODO set configuration should be removed when we finish implement header parser */
-		mInputDataSource->setSampleRate(sampleRate);
-		mInputDataSource->setChannels(channels);
 		medvdbg("size : %u samplerate : %d channels : %d\n", size, sampleRate, channels);
```
Comment (JeongJunSik) [inline framework/src/media/InputHandler.cpp]: *size

Comment (junmin-kim) [reply framework/src/media/InputHandler.cpp]: Fixed it

**Comment by seinfra** [comment]
Target : [8578eab3bf5d954c9555bb17fdbf43b86a80f2df] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [8578eab3bf5d954c9555bb17fdbf43b86a80f2df] - Code Rule Check (C++) OK.

**Pull request merged**